steampy/market.py

```diff
@@ -52,6 +52,20 @@ def get_my_market_listings(self) -> dict:
         listings = get_market_listings_from_html(response.text)
         listings = merge_items_with_descriptions_from_listing(listings, listing_id_to_assets_address,
                                                               assets_descriptions)
+        n_showing = int(text_between(response.text, '<span id="tabContentsMyActiveMarketListings_end">', '</span>'))
+        n_total = int(text_between(response.text, '<span id="tabContentsMyActiveMarketListings_total">', '</span>'))
```

Comment:
n_showing is the number of items actually shown (I prefer get them from the page than to hardcode 10),
n_total is the number of items you are selling